js/functions.js

```diff
@@ -51,6 +51,9 @@ function addElement (row, minimalLength, addSymbol) {
   return row;
 }
 
-export {addElement};*/
+export {addElement};
```

Comment:
Если решили делать экспорт в конце файла, то давай объявлять его один раз

```suggestion
export {
    isEscapeKey,
    addElement,
};
```